src/sdk/accountspace_fs.js

```diff
@@ -494,6 +494,16 @@ class AccountSpaceFS {
         return data;
      }
 
+     async _get_account_decrypted_data_optional_if_exists(account_path, should_decrypt_secret_key) {
```

Comment:
@shirady looks good, can you please add jsdoc? I think documenting this function is very important

Reply:
I didn't add JSDoc in this file because I was afraid it would look messy.
I would add JSDoc for the functions in this file (at least the internal ones) in a different PR.

Reply:
I added for this case in the meantime.

Reply:
I would add a comment about the importance of this function in concurency

Reply:
I added notes for the function, I hope it is clearer.